ts/adaptors/lite/Parser.ts

```diff
@@ -189,7 +189,7 @@ export class LiteParser implements MinDOMParser<LiteDocument> {
         //
         // Get the child to be added to the node
         //
-        const kind = tag.match(/<(.*?)[\s\n>]/)[1].toLowerCase();
+        const kind = tag.match(/<(.*?)[\s\n>/]/)[1].toLowerCase();
```

Comment:
Should that not be 
```suggestion
        const kind = tag.match(/<(.*?)[\s\n/>]/)[1].toLowerCase();
```
?

Reply:
The order of the characters in the `[...]` doesn't matter.  It just matches up to any one of the enclosed characters.  So it is correct as is.

Reply:
Right. I now realise that it is only any of the four characters.